src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -77,6 +82,9 @@ public bool ExecuteCommand(FormatDocumentCommandArgs args, CommandExecutionConte
                 return false;
             }
 
+            var cancellationToken = context.OperationContext.UserCancellationToken;
```

Comment:
this is wrong. you should use "c", not context.OperationContext.UserCancellation since you took over ownership - OperationContent.TaskOwnership.

Reply:
actually not c but c.CancellationToken

Reply:
OK, fixed that.